modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -0,0 +1,154 @@
+= Provision a Serverless Cluster
+:description: Learn how to create a Serverless Cloud cluster.
+:page-cloud: true
+:page-beta: true
+
+Redpanda Serverless is the easiest way to test event streaming in the cloud: You can start streaming 30 seconds after clicking to create the cluster. 
```

Comment:
Easiest way to *start* with event streaming. Not just to test it. Serverless is a production-worthy choice. It isn't limited to testing only. Smaller workloads stay on serverless forever. Larger migrate to Dedicated/BYOC as they grow.

Start streaming in 30 seconds. Fullstop.